.github/workflows/ui-pr-build.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
-        cache-dependency-path: '**/package-lock.json'
+        cache-dependency-path: 'src/UI/package-lock.json'
```

Comment:
Why has this changed?

Reply:
There was a package.lock.json file outside the src folder that I deleted. I thought its because I installed it in the wrong place. Was that a mistake?